apps/web/src/app/(main)/sheet/[moduleId]/page.tsx

```
@@ -0,0 +1,213 @@
+            </div>
+          </div>
+          <h1 className="text-3xl font-bold text-white mb-2">
+            {currentModule.name}
+          </h1>
+        </div>
+
+        {/* Content */}
+        <div className="bg-ox-content rounded-lg p-8 prose prose-invert max-w-none font-DMfont border border-ox-header">
+          <div
+            // eslint-disable-next-line react/no-danger -- Safe: docContent is sanitized with DOMPurify before rendering
+            dangerouslySetInnerHTML={{ __html: sanitizedDocContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)